doc/api/assert.md

```diff
@@ -114,7 +114,7 @@ assert.deepEqual([[[1, 2, 3]], 4, 5], [[[1, 2, '3']], 4, 5]);
 //   ]
 ```
 
-To deactivate the colors, use the `NODE_DISABLE_COLORS` environment variable.
+To deactivate the colors, use `NO_COLOR` or `NODE_DISABLE_COLORS` environment variables.
```

Comment:
I think we should keep the `the`? And I think it's singular because we only want to use either or, not both at the same time? CC: @Trott